graphql/handler/transport/websocket.go

```diff
@@ -232,25 +232,27 @@ func (c *wsConnection) subscribe(start time.Time, message *operationMessage) {
 	go func() {
 		defer func() {
 			if r := recover(); r != nil {
-				userErr := rc.Recover(ctx, r)
-				c.sendError(message.ID, &gqlerror.Error{Message: userErr.Error()})
+				gqlerr := &gqlerror.Error{}
+				if err := rc.Recover(ctx, r); err != nil {
+					gqlerr.Message = err.Error()
```

Comment:
Is there a reason you're throwing `err` away here? it's already a `gqlerror` at this point, and will have potentially useful information like `Path` already set.

Reply:
The existing code was doing the same, anyway I've update the PR to keep existing gqlerr in place.